### PR TITLE
multiple slot drag handling

### PR DESCRIPTION
**Pull request by Sczlog**
1. add support for multiple slot dragging by divide component by its slot.

- single slot nested drop handling

https://user-images.githubusercontent.com/29588646/148869916-93f7e02a-3877-492e-b2db-a5466de46cb2.mp4

- multiple slot drop handling

https://user-images.githubusercontent.com/29588646/148869940-6d57ce41-16fd-4b7a-8292-2e37fc42ab87.mp4

### CODE REVIEW

File: packages/editor/src/EditorStore.ts [hunk 1]
```diff
@@ -43,23 +43,22 @@ class EditorStore {
   }
 
   get dragOverComponentId() {
```
Comment (tanbowensg) [inline packages/editor/src/EditorStore.ts:45]: add comments here

Comment (Sczlog) [reply packages/editor/src/EditorStore.ts:54]: added

**Comment by Yuyz0112** [comment]
Could you post a demo video of this new feature? @Sczlog

**Comment by Sczlog** [comment]
@Yuyz0112 demo updated

**Review by tanbowensg — APPROVED** [review]

**Pull request merged**